docs/content/manual/release-specific/v1.1.0/rwx_feature.md

```diff
@@ -0,0 +1,255 @@
+
+# RWX volume with Toleration.
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Set some Toleration.
+5. Verify the ShareManager pods have the toleration and annotation updated.
+
+# Detach/Delete operation on an RWX volume.
+1. Detach action on the Longhorn UI should not work on RWX volume.
+2. On deletion of the RWX volume, the ShareManager CRDs should also get deleted.
+
+# Crash instance e manager of the RWX volume
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Crash the instance manager.
+5. On crashing the IM, the ShareManager pods should be immediately redeployed.
+5. Based on the setting `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly`, the workload pods will get redeployed.
+6. On recreating on workload pods, the volume should get attached successfully.
+7. If `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly` is disabled, user should see I/O error on the mounted point.
+
+# Reboot the ShareManager and workload node
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Reboot the ShareManager node.
+5. The ShareManager pod should move to another node.
+6. As the instance e manager is on the same node and based on setting `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly`, the workload should be redeployed and volume should be available to user.
+7. Reboot the workload node.
+8. On restart on the node, pods should get attached to the volume. Verify the data.
+
+# Power down the ShareManager and workload node.
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Power down the ShareManager node.
+5. The ShareManager pod should move to another node.
+6. As the instance manager is on the same node and based on the setting `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly`, the workload should be redeployed and volume should be available to user.
+7. Power down the workload node.
+8. The workload pods should move to another node based on `Pod Deletion Policy When Node is Down` setting.
+9. Once the pods are up, they should get attached to the volume. Verify the data.
+
+# Kill the nfs process in the ShareManager
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Kill the NFS server in the ShareManager pod.
+5. The NFS server should retry to come up.
```

Comment:
@joshimoo Please correct this if it's wrong.

Reply:
removed the retry instead the pod will go into terminate then restarted by the sharemanager controller.
the ganesha service pid can be seen by `cat /var/run/ganesha.pid`